docs/dev/hardware.rst

```diff
@@ -30,6 +30,9 @@ the output of the following command (on the target device), so the autoupdater
 can work::
 
     lua -e 'print(require("platform_info").get_image_name())'
+    
+Best practise is to generate an image first by using a parameter at will and then
```

Comment:
I'd mention the objective first, as is done with the second parameter below.
> Best practice to get the first parameter is …

Reply:
This should be clear in the context, I guess. Because that is explained in the paragraph before.

Reply:
Ah yes, I missed that.

Reply:
> by using an arbitrary value
instead of
> by using a parameter at will

- "at will" seems more idiomatic to me
- also we should be talking about the value of the parameter I guess.